functions_and_scope_lab.js

```diff
@@ -0,0 +1,303 @@
+// JOSEPH P. PASAOA
+//
+
+
+// function declarations
+
+const spacer = () => {console.log("\n")}
+
+// f.1
+const whosLarger = (num1, num2) => {
+  if (num1 > num2) {
+    return num1;
+  } else {
+    return num2;
+  }
+}
+
+// f.2
+const getDogAge = (num1) => num1 * 7
+
```

Comment:
Remember to add return.

Reply:
For f.2? I thought implicit returns don't require them